DepositeCalcTests/Pages/LoginPage.cs

```diff
@@ -39,4 +41,75 @@ public bool IsOpened()
             return isLoginFldPresent;
         }
     }
+
+    public class RemindPasswordView
+    {
+        private IWebDriver driver;
+        private IWebElement openButton;
+        public RemindPasswordView(IWebDriver driver, IWebElement openButton)
+        {
+            this.driver = driver;
+            this.openButton = openButton;
+        }
+
+        private IWebElement RemindPasswordForm => driver.FindElement(By.XPath("//iframe[@id='remindPasswordView']"));
```

Comment:
1. you can change By to 'By.Id'
2. you can extract frame id to a separate variable and re-use later.

```suggestion
        private string _frameId = "remindPasswordView";
        private IWebElement RemindPasswordForm => driver.FindElement(By.Id(_frameId));
```